rspace/src/main/scala/coop/rchain/rspace/nextgenrspace/history/Store.scala

```diff
@@ -98,9 +99,30 @@ final case class LMDBStore[F[_]: Sync] private[history] (
       }
     }
 
+  @SuppressWarnings(Array("org.wartremover.warts.Throw"))
```

Comment:
:-(

Reply:
yeah I know. any thoughts how to improve?